src/Components/RoundedCards/RoundedCards.jsx

```
@@ -1,29 +1,31 @@
         </p>
       </div>
       <div className="lowerSec">
         {data.map((e) => (
-          <div className="lowerCard" key={e.id}>
-            <img src={e.img} alt="card" />
-            <h4>{e.title}</h4>
-            <p>
-              {e.desc}
-            </p>
-          </div>
+          <Link to={e.link}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)